src/Esfa.Vacancy.Manage.Api/App_Start/StructuremapConfig.cs

```diff
@@ -52,6 +53,8 @@ public static void Start()
             GlobalConfiguration.Configuration.DependencyResolver = new StructureMapWebApiDependencyResolver(container);
             DependencyResolver.SetResolver(StructureMapDependencyScope);
             DynamicModuleUtility.RegisterModule(typeof(StructureMapScopeModule));
+
+            Debug.WriteLine(container.WhatDidIScan());
```

Comment:
Is this required?

Reply:
whoops, didn't realise that was still in the branch I've PR'd.